server/core/config.cc

```diff
@@ -461,6 +462,18 @@ config::ParamString Config::s_admin_ssl_key(
     "Admin SSL key",
     "");
 
+config::ParamEnum<admin_ssl_version_t> Config::s_admin_ssl_version(
+    &Config::s_specification,
+    CN_ADMIN_SSL_VERSION,
+    "Admin allowed SSL version",
+    {
+        {ADMIN_SSL_VERSION_TLS10, "TLSv1.0"},
```

Comment:
Use the same values as defined here: https://mariadb.com/kb/en/mariadb-maxscale-25-mariadb-maxscale-configuration-guide/#ssl_version

In addition, the default value should be in the configuration parameter enum as it should be possible to explicitly define the default value. The default can be added as the `MAX` value as it supports the highest available TLS version that both the client and the server support.